org.jacoco.build/pom.xml

```diff
@@ -454,8 +454,7 @@
         <plugin>
           <groupId>org.codehaus.mojo</groupId>
           <artifactId>exec-maven-plugin</artifactId>
-          <!--  latest version which runs with Java 5 -->
```

Comment:
As a note - this comment is outdated since the time we require Java version greater than 5 to start Maven.

Reply:
Ok, thanks for this hint!